source/macos/AdaptiveCards/AdaptiveCards/Views/ACRMultilineInputTextView.swift

```diff
@@ -7,10 +7,12 @@ class ACRMultilineInputTextView: NSView, NSTextViewDelegate {
     @IBOutlet var textView: ACRTextView!
     
     private var placeholderAttrString: NSAttributedString?
+    private var config: InputFieldConfig
     var maxLen: Int = 0
```

Comment:
```suggestion
    private let config: InputFieldConfig
    var maxLen: Int = 0
```
This can be a `let` property?